corehq/apps/userreports/tests/test_expressions.py

```diff
@@ -807,3 +811,44 @@ def test_unsupported_evluator_statements(self, eq, context):
         "context_variables": context
     })
     self.assertEqual(expression({}), None)
+
+
+class TestFormsExpressionSpec(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.domain = uuid.uuid4().hex
+        factory = CaseFactory(domain=cls.domain)
+        [cls.case] = factory.create_or_update_case(CaseStructure())
+        cls.forms = [f.to_json() for f in FormAccessors(cls.domain).get_forms(cls.case.xform_ids)]
```

Comment:
I think it'd be worth adding a test that ensures a different form in the domain is not also included in the results